services/community-site/src/assets/icons/greenCircularCheck.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+interface CircleCheckProps {
+  color?: string;
+}
+
+const GreenCircledCheckIcon = (props: CircleCheckProps) => {
```

Comment:
This is not exported. Shouldn`t we export both of them or separate them into two files to use the same pattern.